src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -52,6 +52,13 @@ internal partial class XamlFileGenerator
 
 		private static readonly string[] FrameworkTemplateTypes = new[] { "DataTemplate", "ControlTemplate", "ItemsPanelTemplate" };
 
+		private static readonly XamlType _elementStubXamlType = new XamlType(
```

Comment:
@MartinZikmund Did you ensure that they are not mutated in any code path? If they are mutated, this will be a very hard-to-investigate bug :)